tests/test_settings.py

```diff
@@ -63,12 +68,14 @@ def test_toml_override(self, mock_getcwd):
     @mock.patch('semantic_release.settings.getcwd', return_value='/tmp/')
     def test_no_raise_toml_error(self, mock_getcwd, mock_debug):
         # create temporary toml config file
-        dummy_conf_path = '/tmp/pyproject.toml'
+        dummy_conf_path = os.path.join(here, 'tmp', 'pyproject.toml')
```

Comment:
This change is causing the test to fail.

Reply:
Same comment as above.